picasso-pollexor/src/main/java/com/squareup/picasso3/pollexor/PollexorRequestTransformer.kt

```diff
@@ -0,0 +1,77 @@
+package com.squareup.picasso3.pollexor
+
+import android.net.Uri
+import android.os.Build.VERSION
+import android.os.Build.VERSION_CODES
+import com.squareup.pollexor.Thumbor
+import com.squareup.picasso3.Picasso.RequestTransformer
+import com.squareup.picasso3.Request
+import com.squareup.pollexor.ThumborUrlBuilder
+import com.squareup.pollexor.ThumborUrlBuilder.ImageFormat.WEBP
+import java.lang.IllegalArgumentException
+
+/**
+ * A [RequestTransformer] that changes requests to use [Thumbor] for some remote
+ * transformations.
+ * By default images are only transformed with Thumbor if they have a size set,
+ * unless alwaysTransform is set to true
+ */
+class PollexorRequestTransformer @JvmOverloads constructor(
```

Comment:
I think you'll need to declare constructor overloads manually here, `@JvmOverloads` will only give you

```
PollexorRequestTransformer(thumbor)
PollexorRequestTransformer(thumbor, alwaysTransform)
PollexorRequestTransformer(thumbor, alwaysTransform, callback)
```

while Java version also has

```
PollexorRequestTransformer(thumbor, callback)
```

Smth like Japicmp would've likely caught this.